crates/starknet/src/starknet/mod.rs

```diff
@@ -163,6 +163,7 @@ impl Starknet {
     // Initialize values for new pending block
     pub(crate) fn generate_pending_block(&mut self) -> DevnetResult<()> {
         Self::update_block_context(&mut self.block_context);
+        self.update_pending_block_timestamp();
         self.restart_pending_block()?;
```

Comment:
update_pending_block_timestamp modifies the block header, then restart_pending_block also modifies the block header. This will lead to overriding the desired expected offset

Reply:
Yes, but this is what we want to do right? Or do you have another idea to do it? I'm not sure if it's just a comment or if you want me to take any actions.

Reply:
is it still relevant after changes in 58460a7974dfef8f37839b83789402e124b682d4 and the merge of  https://github.com/0xSpaceShard/starknet-devnet-rs/pull/235 PR?

Reply:
no, the comment is not relevant